packages/gen-ai/bff/build.openshift.sh

```
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Configuration
+DOCKERFILE="Dockerfile"
+
+# Get Git information
+# Get the remote and branch from upstream
+UPSTREAM=$(git rev-parse --abbrev-ref @{u} 2>/dev/null || echo "")
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)